src/EnergyPlus/UFADManager.cc

```diff
@@ -384,6 +341,48 @@ namespace RoomAir {
         }
     }
 
+    Real64 sumUFADConvGainPerPlume(EnergyPlusData &state, int const zoneNum, Real64 const numOccupants)
+    {
+        Real64 zoneElecConv(0.0); // zone elec equip design convective gain [W]
+        for (auto const &zoneElectric : state.dataHeatBal->ZoneElectric) {
+            if (zoneElectric.ZonePtr == zoneNum) {
+                zoneElecConv += zoneElectric.DesignLevel * zoneElectric.FractionConvected;
+            }
+        }
+
+        Real64 zoneGasConv(0.0); // zone gas equip design convective gain [W]
+        for (auto const &zoneGas : state.dataHeatBal->ZoneGas) {
+            if (zoneGas.ZonePtr == zoneNum) {
+                zoneGasConv += zoneGas.DesignLevel * zoneGas.FractionConvected;
+            }
+        }
+
+        Real64 zoneOthEqConv(0.0); // zone other equip design convective gain [W]
+        for (auto const &zoneOtherEq : state.dataHeatBal->ZoneOtherEq) {
+            if (zoneOtherEq.ZonePtr == zoneNum) {
+                zoneOthEqConv += zoneOtherEq.DesignLevel * zoneOtherEq.FractionConvected;
+            }
+        }
+
+        Real64 zoneHWEqConv(0.0); // zone hot water equip design convective gain [W]
+        for (auto const &zoneHWEq : state.dataHeatBal->ZoneHWEq) {
+            if (zoneHWEq.ZonePtr == zoneNum) {
+                zoneHWEqConv += zoneHWEq.DesignLevel * zoneHWEq.FractionConvected;
+            }
+        }
+
+        Real64 zoneSteamEqConv(0.0); // zone steam equip design convective gain [W]
+        for (auto const &zoneSteamEq : state.dataHeatBal->ZoneSteamEq) {
+            if (zoneSteamEq.ZonePtr == zoneNum) {
+                zoneSteamEqConv += zoneSteamEq.DesignLevel * zoneSteamEq.FractionConvected;
+            }
+        }
+
+        Real64 numPlumes = (numOccupants > 0.0) ? numOccupants : 1.0;
+
+        return (numOccupants * 73.0 + zoneElecConv + zoneGasConv + zoneOthEqConv + zoneHWEqConv + zoneSteamEqConv) / numPlumes;
```

Comment:
It would be nice if the 73 here was a constexpr named variable, rather than just a random magic number.  No need for extra commits, but something to think about in future fixes.